atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/timestamp/InDbTimestampBoundStore.java

```diff
@@ -197,8 +198,20 @@ private void createLimit(Connection connection, long limit) throws SQLException
 
     private void createTimestampTable(Connection connection) throws SQLException {
         try (Statement statement = connection.createStatement()) {
-            statement.execute(String.format("CREATE TABLE IF NOT EXISTS %s ( last_allocated int8 NOT NULL )",
-                    timestampTable.getQualifiedName()));
+            if (getDbType(connection).equals(DBType.ORACLE)) {
+                try {
+                    statement.execute(String.format("CREATE TABLE %s ( last_allocated NUMBER(38) NOT NULL )",
```

Comment:
Do you want to see if `BIGINT` works due to the connector? If not, `NUMBER(19, 0)` is closer to what we want (Many websites have it listed as roughly equivalent: http://weblogs.sqlteam.com/jamesw/archive/2010/08/04/datatypes-translation-between-oracle-and-sql-server-part-2-number.aspx)